runtime/databricks/automl_runtime/forecast/prophet/model.py

```diff
@@ -61,19 +64,35 @@ class ProphetModel(mlflow.pyfunc.PythonModel):
     """
```

Comment:
Do we need to update the version of `databricks-automl-runtime` in `PROPHET_CONDA_ENV`?